### PR TITLE
D8UN-554 Fiscal Comm: Add twitter icon to footer

### PR DESCRIPTION
**Pull request by ooneill547**

### CODE REVIEW

File: web/sites/fiscalcommissionni/themes/fiscalcommissionni_theme/templates/block/block--contactus.html.twig [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  {% if label %}
+    <h2{{ title_attributes }}>{{ label }}</h2>
+  {% endif %}
+  {{ title_suffix }}
+  {% block content %}
+    <div{{ content_attributes.addClass('content') }}>
+      {{ content }}
+
+      <div class="social-icons">
+        <a href="https://twitter.com/fiscal_ni" target="_blank">
```
Comment (omahm) [inline web/sites/fiscalcommissionni/themes/fiscalcommissionni_theme/templates/block/block--contactus.html.twig:50]: Add rel="noopener noreferrer" to the link. See https://medium.com/sedeo/how-to-fix-target-blank-a-security-and-performance-issue-in-web-pages-2118eba1ce2f

**Review by omahm — CHANGES_REQUESTED** [review]
LG, just a comment on security of _blank.

**Comment by ooneill547** [comment]
I added the noopener and noreferrer to the link in fiscal comm and also added these to the link in uregni that acts the same

**Review by omahm — APPROVED** [review]
LGTM

**Pull request merged**